backend/app/Http/Controllers/CategoryController.php

```diff
@@ -14,7 +14,10 @@ class CategoryController extends Controller
      */
     public function index()
     {
-        return Category::all();
+        $categories = Category::all();
+        return response([
+            'categories' => $categories,
+        ], 201);
```

Comment:
```suggestion
        return  response()->json([
            'categories' => $categories,
        ], 201);
```

Please use ` response()->json()` when returning the data

Reply:
This applies to all return response